### PR TITLE
Fix: cash cassettes fixes

### PR DESCRIPTION
**Pull request by hellypi**

### CODE REVIEW

**Comment by hellypi** [comment]
[Cash cassettes fixes](https://trello.com/c/DVOJangP/581-cash-cassettes-fixes)

File: new-lamassu-admin/src/pages/Maintenance/CashCassettes.js [hunk 1]
```diff
@@ -51,6 +52,16 @@ const GET_MACHINES_AND_CONFIG = gql`
...
+/* 
+  // for cash in total calculation
+  bills {
+    fiat
+    deviceId
+    created
+    cashbox
+  }
+*/
+
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Maintenance/CashCassettes.js:64]: Is this comment still necessary?

File: new-lamassu-admin/src/pages/Maintenance/CashCassettesFooter.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import { makeStyles } from '@material-ui/core'
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Maintenance/CashCassettesFooter.js:1]: Couple of comments spread throughout this file

File: new-lamassu-admin/src/pages/Maintenance/CashCassettesFooter.styles.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    bottom: 0,
+    width: '100vw',
+    backgroundColor: 'white',
+    display: 'flex',
+    justifyContent: 'space-around',
+    boxShadow: [[0, -1, 10, 0, 'rgba(50, 50, 50, 0.1)']]
+  },
+  flex: {
+    display: 'flex',
+    // temp marginLeft until cashIn square is enabled
```
Comment (chaotixkilla) [inline new-lamassu-admin/src/pages/Maintenance/CashCassettesFooter.styles.js:28]: Same here

**Review by chaotixkilla — CHANGES_REQUESTED** [review]
Pretty much only forgotten comments

**Comment by hellypi** [comment]
rebased to fix conflict

**Review by chaotixkilla — APPROVED** [review]

File: lib/new-admin/bills.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    ORDER BY device_id, created DESC`
+  )
+    .then(res => {
+      return res.map(item => ({
+        fiat: item.fiat,
+        deviceId: item.device_id,
+        cashbox: item.cashbox,
+        created: item.created
+      }))
+    })
```
Comment (RafaelTaranto) [inline lib/new-admin/bills.js]: We don't want to leave this query here. Either remove everything related to it or make it return a error, devs should not be allowed to use this.

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**